Mega Man prototype stuff

srl447 opened the pull request:
It's shooting script and the scene I made for the prototype. Also, the starting work for enemies.

srl447 committed:
Merge remote-tracking branch 'origin/master' into Sam

srl447 committed:
Mega Man Start

All the stuff I programmed in like 12 minutes

thaotnguyen commented:
Can you remove the files that aren't necessary? A lot of these look unrelated and people get really confused by them.

srl447 commented:
I don't know how to remove files now actually lol. The only things I don't know what they are are like the last 4 files, and idk if they're actually relevant to how I changed the project settings. Most of the files are just me moving all the old files to a new folder

srl447 committed:
Messing with Jumping

Also, added temp art to make me not go crazy from squares, fixing the trail renderer, and gitignoring .vs

srl447 committed:
ignoring obj folder

srl447 committed:
Ignoring obj folder
